docs/user-guide/binned-data/histogramming-grouping-and-binning.ipynb

```diff
@@ -0,0 +1,251 @@
+    "plot(ds)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the above plot we can only see a single line, since the three solutions yield exactly the same result (neglecting floating-point rounding errors):\n",
+    "\n",
+    "- `histogram` sorts data points into 'x' bins, summing immediately.\n",
+    "- `groupby` groups by 'x' and then on-the-fly sums all data points falling in the same 'x' bin.\n",
+    "- `bin` sorts data points into 'x' bins.\n",
+    "  Summing all rows in a bin yields the same result as grouping and summing directly.\n",
+    "\n",
+    "So in this case we get equivalent results, but the application areas differ, as described in more detail in the following sections.\n",
+    "\n",
+    "## Histogramming\n",
+    "\n",
+    "`histogram` directly sums the data and is efficient.\n",
+    "Limitations are:\n",
+    "\n",
+    "- Can currently only histogram along a single dimension.\n",
+    "- Can currently only apply the \"sum\" to accumulate into a bin.\n",
+    "  Support for \"mean\" operations is considered for the future.\n",
+    "  \n",
+    "While histogramming is only supported along a single dimension, we *can* histogram binned data (since [binning preserves the `'y'` coord](#Binning)), to create 2-D (or N-D) histograms:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "binned = sc.bin(data, edges=[xbins])\n",
+    "ybins = sc.Variable(dims=['y'], unit=sc.units.m, values=np.linspace(0,1,num=30))\n",
+    "hist = sc.histogram(binned.bins, ybins)\n",
+    "plot(hist)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hist"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Another capability of `histogram` is to histogram a dimension that has previously been binned, with a different or higher resolution, i.e., different bin edges.\n",
+    "Compare to the plot of the initial example:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "binned = sc.bin(data, edges=[xbins])\n",
+    "xbins_fine = sc.Variable(dims=['x'], unit=sc.units.m, values=np.linspace(0,1,num=100))\n",
+    "plot(sc.histogram(binned.bins, xbins_fine))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Grouping\n",
+    "\n",
+    "`groupby` is more flexible in terms of operations than can be applied and may be the go-to solution when a quick one-liner is required.\n",
+    "Limitations are:\n",
+    "\n",
+    "- Can only group along a single dimension.\n",
+    "- Works best for small to medium-sized data, or if data is mostly sorting along the grouping dimension.\n",
+    "  Slow if millions of small subgroups contribute to each group.\n",
```

Comment:
Not sure if "subgroups" is very clear here... We are talking about groups and suddenly we bring subgroups into the picture, without knowing what they are.